tests/spread/rockcraft/extension-expressjs/task.yaml

```diff
@@ -0,0 +1,72 @@
+summary: expressjs extension test
+environment:
+  SCENARIO/bare: bare
+  SCENARIO/base_2404: ubuntu-24.04
+  ROCKCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS: "true"
+
+execute: |
+  NAME="expressjs-${SCENARIO//./-}"
+  ROCK_FILE="${NAME}_0.1_amd64.rock"
+  IMAGE="${NAME}:0.1"
+
+  sudo apt-get update -y && sudo apt-get install npm -y
+  npm install -g express-generator@4
+  express app
+  cd app && npm install && cd ..
+
+  run_rockcraft init --name "${NAME}" --profile expressjs-framework
+  sed -i "s/^name: .*/name: ${NAME}/g" rockcraft.yaml
+  sed -i "s/^base: .*/base: ${SCENARIO//-/@}/g" rockcraft.yaml
+  if [ "${SCENARIO}" != "bare" ]; then
+    sed -i "s/^build-base: .*/build-base: ${SCENARIO//-/@}/g" rockcraft.yaml
```

Comment:
this is a bit weird - where does the original `build-base`, that you're replacing here, come from?

Reply:
Updated the template to be by default: bare base
If the scenario is not bare, it will change the build-base to match the scenario's build base from templates embedded 24.04 base.
Thank you!

Reply:
Actually, I'm wondering what the default build base should be in this case - bare or 24.04.

The reason for using the bare as default is to be **production ready** (with a small image with less surface area)
The reason for using the 24.04 as default is **for the tutorial** where the initialization tool defaults to 24.04 and we guide them to using the slimmed version of the image (by switching to bare).

What do you think should be the default? I think we could use the bare base and start the tutorial by changing the base to 24.04, although that might seem a little weird until the user finishes all the tutorial steps.

Perhaps @erinecon and @jdkandersson would also be interested in this discussion point :)